smart-contract-verifier/src/settings.rs

```diff
@@ -68,6 +69,29 @@ impl Default for SoliditySettings {
     }
 }
 
+#[derive(Debug, Clone, PartialEq, Eq, Deserialize)]
+#[serde(default, deny_unknown_fields)]
+pub struct VyperSettings {
+    pub enabled: bool,
+    pub compilers_dir: PathBuf,
+    #[serde(with = "serde_with::rust::display_fromstr")]
+    pub refresh_versions_schedule: Schedule,
+    pub list_url: Url,
```

Comment:
I suggest to make it the same way it is done for `SoliditySettings`: to make it through `pub fetcher: FetcherSettings` field